templates/xfs_quota.epp

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
fyi... I did not really dive deep into this bash script. I trust it works as expected and did not see any obvious issues.